src/code.c

```diff
@@ -114,8 +114,8 @@ FILE *oldf;
   
 void OpenFile( FILE **fpp, char *name, char * ext, char * identity )
 {
-char bufname[200];
-char buf[200];
+char bufname[300];
```

Comment:
Should 300 be a global constant instead, in case we have to update it in the future?

Reply:
Also ditto for following instances of 300